android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateUtils.java

```diff
@@ -0,0 +1,46 @@
+package com.microsoft.codepush.react;
+
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.WritableMap;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CodePushUpdateUtils {
+
+    public static void copyNecessaryFilesFromCurrentPackage(String diffManifestFilePath, String currentPackageFolderPath, String newPackageFolderPath) throws IOException{
```

Comment:
What about "processDiffUpdate" or something similar? Since you moved this into an update-specific utils, I like the idea of having the method names be as specific to the scenario as possible.